packages/react-hooks/src/useLenientThresholdPercentage.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright 2017-2023 @polkadot/react-hooks authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import type { Perquintill } from '@polkadot/types/interfaces/runtime';
+
+import getCommitteeManagement from '@polkadot/react-api/getCommitteeManagement';
+import { BN, BN_QUINTILL } from '@polkadot/util';
+
+import { createNamedHook } from './createNamedHook.js';
```

Comment:
Tbh, I've just followed the convention.
I think it might be about readability of error pop-up / boundary messages. I'm not sure if harmful, but didn't bite me yet :sweat_smile: 

![image](https://github.com/Cardinal-Cryptography/apps/assets/35614528/38d26952-eac6-4674-b8a3-920699249cd2)
![image](https://github.com/Cardinal-Cryptography/apps/assets/35614528/b59c0c9b-2079-48c0-87f0-03cfa636020b)

vs

![image](https://github.com/Cardinal-Cryptography/apps/assets/35614528/95af6efc-1474-45cd-8b7c-8bec08dc13c5)
![image](https://github.com/Cardinal-Cryptography/apps/assets/35614528/3c797541-d9e5-49c0-951e-0d0b08de29e6)

Reply:
I'll remove it :+1:

Reply:
> it might be about readability of error pop-up / boundary messages

Hmm that might be it, but then the stringification should appear in the error boundary itself - the hook in which it originates from is **very** early.